core/libp2p/protocol/identify/identify_msg_processor.cpp

```diff
@@ -326,7 +326,13 @@ namespace libp2p::protocol {
     }
 
     // memorize the addresses
-    switch (conn_manager_.connectedness(peer_id)) {
+    auto addresses = addr_repo.getAddresses(peer_id);
+    if(!addresses) {
+      log_->error("can not get addresses for peer {}", peer_id.toBase58());
```

Comment:
We decided not to use logger in Libp2p, so better not to add new uses of it

Reply:
there is a separate task to remove logger